Makefile

```diff
@@ -195,46 +195,39 @@ ko-targets = scorecard-ko cron-controller-ko cron-worker-ko cron-cii-worker-ko c
 ko-build-everything: $(ko-targets)
```

Comment:
We need something to test in CI so that it doesn't push to the registry. Would this change work with that? We don't have access from the GH to GCP so all the container builds happen only in GCP with cloudbuild

Reply:
If this is intended just for testing that `ko` builds work, then `--push=false` is probably what you want (and not `--local`, unless you want to make the resulting image available in the Docker daemon, which it looks like you don't)

I've added `--push=false` to all of these for now, and reverted `cloudbuild/scorecard.yaml` since it won't actually push your image now. We can figure out how we want to integrate `ko publish` into your actual releases in another PR.